constants.py

```diff
@@ -7,3 +7,4 @@
 SEED_NODE_TESTNET = ("205.186.154.163", 28467)
 KSIZE = 20
 ALPHA = 3
+TRANSACTION_FEE = 10000
```

Comment:
is this satoshis? millibits? bitcoins? ;)

It'd be nice to have the unit in the name of the "constant"